samples/kvsWebRTCClientMaster.c

```diff
@@ -199,6 +199,12 @@ PVOID sendVideoPackets(PVOID args)
         goto CleanUp;
     }
 
+    MUTEX_LOCK(pSampleConfiguration->sampleConfigurationObjLock);
```

Comment:
There are couple of issues with this approach.

1) This block is replicated for audio and video.
2) This only works for the stock master and not GStreamer based master.

Recommend the following.

The sample applications spin up two threads per peer connection to mimmic the media pipeline. 

1) The thread entry routine which is set in the sample configuration which is set by the appropriate sample should be wrapped by a common routine that handles things like awaiting for the connected state (this block) before calling directly the actual routine.
2) Longer term - spin a single thread that will service all of audio and video for all peer connections.

Reply:
I've created another routine to spin up the video and audio thread. I agree that we should have a single thread to serve all audio and video. But, this will be addressed in a different PR as it is outside the scope.